src/components/new-request/stats.vue

```diff
@@ -58,26 +58,30 @@
 </template>
 
 <script lang="ts">
-import newReqModule from '@/store/new-request-module'
+import { Vue, Component, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
 import { StatsI } from '@/interfaces'
-import { Vue, Component } from 'vue-property-decorator'
 import { getFeatureFlag } from '@/plugins'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
 
 @Component({})
 export default class Stats extends Vue {
+  // Global Getters
+  @Getter getStats!: StatsI
+
+  // Global Actions
+  @Action setStats!: ActionBindingIF
+
   created (): void {
     if (getFeatureFlag('hardcoded_regular_wait_time') === 0 ||
         getFeatureFlag('hardcoded_priority_wait_time') === 0) {
-      newReqModule.getStats()
+      this.setStats(null)
     }
   }
 
-  get stats (): StatsI {
-    return newReqModule.stats
-  }
-
   get autoApprovedCount (): string | number {
-    return (this.stats?.auto_approved_count || '0')
+    return (this.getStats?.data?.auto_approved_count ?? '-')
```

Comment:
I think you mean `||` not `??`. See also lines 93 and 103 below.

Reply:
No. Nullish coalesce operator is used intentionally so that 0 is displayed (or - if empty).